### DISCUSSION TITLE
命令行下载

### DISCUSSION THREAD

**Discussion by ghost** [lifegpc/bili#53] [Q&A] [answered]
现在有了这个下载很方便。但是尝试用start.exe “url" 下载，不能下载。

希望能有这种下载方式的选项，如果看不懂我在说啥，或者会很麻烦，请忽略。

**Comment by lifegpc** [comment, marked as answer]
使用 ```start.exe -i "url"```
顺带一提，新的浏览器扩展在准备，到时候只要浏览器菜单右键单击就能够开始下载
